lib/util/git/index.ts

```diff
@@ -215,6 +218,13 @@ export async function setBranchPrefix(branchPrefix: string): Promise<void> {
   }
 }
 
+export function setIgnoredAuthors(ignoredAuthors?: string[]): void {
+  config.ignoredAuthors = [];
+  if (is.array(ignoredAuthors)) {
+    config.ignoredAuthors = ignoredAuthors;
+  }
```

Comment:
```suggestion
  config.ignoredAuthors = ignoredAuthors ?? [];
```

simplify. array validation is done by config validator, so typescript validation is enough